src/bobbit/history.py

```diff
@@ -22,9 +22,9 @@ def search(self, channel, nick=None, pattern=None, limit=None, reverse=False):
         if pattern:
             pattern = re.compile(pattern)
 
-        messages = self.history[channel]
+        messages = self.history.get(channel, collections.deque())
         if reverse:
-            messages = reversed(messages)
+            messages.reverse()
```

Comment:
I don't think we want to perform the reverse in place as that will permanently change the order of the deque in the history.  I think we want to get a new list.